src/main/java/com/stripe/model/SubscriptionSchedule.java

```diff
@@ -0,0 +1,406 @@
+  Map<String, String> metadata;
+
+  /** String representing the object's type. Objects of the same type share the same value. */
+  @SerializedName("object")
+  String object;
+
+  /** Configuration for the subscription schedule's phases. */
+  @SerializedName("phases")
+  List<Phase> phases;
+
+  /**
+   * Time at which the subscription schedule was released. Measured in seconds since the Unix epoch.
+   */
+  @SerializedName("released_at")
+  Long releasedAt;
+
+  /** ID of the subscription once managed by the subscription schedule (if it is released). */
+  @SerializedName("released_subscription")
+  String releasedSubscription;
+
+  /** Behavior of the subscription schedule and underlying subscription when it ends. */
+  @SerializedName("renewal_behavior")
+  String renewalBehavior;
+
+  /**
+   * Interval and duration at which the subscription schedule renews for when it ends if
+   * `renewal_behavior` is `renew`.
+   */
+  @SerializedName("renewal_interval")
+  SubscriptionScheduleRevision.RenewalInterval renewalInterval;
+
+  /** ID of the current revision of the subscription schedule. */
+  @SerializedName("revision")
+  String revision;
+
+  /** Possible values are `not_started`, `active`, `completed`, `released`, and `canceled`. */
+  @SerializedName("status")
+  String status;
+
+  /** ID of the subscription managed by the subscription schedule. */
+  @SerializedName("subscription")
+  @Getter(lombok.AccessLevel.NONE)
+  @Setter(lombok.AccessLevel.NONE)
+  ExpandableField<Subscription> subscription;
+
+  /** Get id of expandable `customer` object. */
+  public String getCustomer() {
+    return (this.customer != null) ? this.customer.getId() : null;
+  }
+
+  public void setCustomer(String id) {
+    this.customer = ApiResource.setExpandableFieldId(id, this.customer);
+  }
+
+  /** Get expanded `customer`. */
+  public Customer getCustomerObject() {
+    return (this.customer != null) ? this.customer.getExpanded() : null;
+  }
+
+  public void setCustomerObject(Customer expandableObject) {
+    this.customer = new ExpandableField<Customer>(expandableObject.getId(), expandableObject);
+  }
+
+  /** Get id of expandable `subscription` object. */
+  public String getSubscription() {
+    return (this.subscription != null) ? this.subscription.getId() : null;
+  }
+
+  public void setSubscription(String id) {
+    this.subscription = ApiResource.setExpandableFieldId(id, this.subscription);
+  }
+
+  /** Get expanded `subscription`. */
+  public Subscription getSubscriptionObject() {
+    return (this.subscription != null) ? this.subscription.getExpanded() : null;
+  }
+
+  public void setSubscriptionObject(Subscription expandableObject) {
+    this.subscription =
+        new ExpandableField<Subscription>(expandableObject.getId(), expandableObject);
+  }
+
+  /** Retrieves the list of your subscription schedules. */
+  public SubscriptionScheduleCollection list(Map<String, Object> params) throws StripeException {
```

Comment:
@ob-stripe totally random but do we really need those duplicate collection classes? (nothing to do with autogen, just realizing I haven't asked in a while)